fdbserver/storageserver.actor.cpp

```diff
@@ -6010,7 +6010,9 @@ ACTOR Future<Void> tryGetRangeFromBlob(PromiseStream<RangeResult> results,
 		tr->reset();
 		tr->setVersion(fetchVersion);
 		tr->trState->taskID = TaskPriority::FetchKeys;
-		wait(tryGetRange(results, tr, keys)); // fail back to storage server
+		fmt::print(
```

Comment:
Should also be a trace event instead of a printf

Reply:
already tracked earlier in this catch block. remove this print